Readme.md

```diff
@@ -17,6 +17,7 @@ Tracee is composed of the following sub-projects:
 - [libbpgo](libbpfgo) - Go library for eBPF programming using Linux's [libbpf](https://github.com/libbpf/libbpf)
 
 ## Getting started
+![High Level Overview](images/highleveloverview.png)
```

Comment:
I wouldn't put this under getting started. The end user shouldn't even be aware of the intricacies of the internal components. In the new readme (#647 ) this would fit under the "components" section

Reply:
Actually, perhaps it would be beneficial to have an architecture.md file, for the sake of introducing contributors to tracee (as opposed to users which have the docs site). at first this doc can just contain this diagram, perhaps in the future we will add content

Reply:
https://matklad.github.io//2021/02/06/ARCHITECTURE.md.html

Reply:
Yeah sure I'm OK with that. I can change it live in architecture.md instead.